src/gridfs-stream/download.ts

```diff
@@ -138,120 +188,122 @@ export class GridFSBucketReadStream extends Readable {
    * @fires GridFSBucketWriteStream#end
    */
 
-  abort(callback: Callback) {
-    var _this = this;
+  abort(callback: Callback<undefined>) {
     this.push(null);
     this.destroyed = true;
     if (this.s.cursor) {
-      this.s.cursor.close((error: any) => {
-        _this.emit('close');
+      this.s.cursor.close((error?: Error) => {
+        this.emit('close');
         callback && callback(error);
       });
     } else {
       if (!this.s.init) {
         // If not initialized, fire close event because we will never
         // get a cursor
-        _this.emit('close');
+        this.emit('close');
       }
       callback && callback();
     }
   }
 }
 
-function throwIfInitialized(self: any) {
+function throwIfInitialized(self: GridFSBucketReadStream) {
   if (self.s.init) {
     throw new Error('You cannot change options after the stream has entered' + 'flowing mode!');
```

Comment:
Not your code I know, but looks like this doesn't need concatenation anymore maybe there used to be a var in between